packages/common/utils/src/minBy.ts

```diff
@@ -7,7 +7,7 @@
  *   // 최솟값을 찾을 배열
  *   collection: T[],
  *   // 배열의 값을 계산하는 방법
- *   iteratee: (element: T) => any
+ *   iteratee: (element: T) => T | undefined
```

Comment:
I think not this should be fixed to 
```suggestion
 *   iteratee: (element: T) => number
```

according to code itself return type of iteratee function is number not T | undefined

Reply:
Sorry, that's my mistake. 
I fixed it according to your suggestion.

[docs: fix iteratee return type](https://github.com/toss/slash/pull/59/commits/dd5395d55ee60624e61e8c9f3bed46ffbc107742)